charters/DataProcessingService/charter.md

```diff
@@ -0,0 +1,80 @@
+
+# Project Name
+HCA Data Processing Service
+
+## Description
+
+The data processing service is a centralized space to perform all transformative functionality to scientific data ingested into the Human Cell Atlas Data Coordination Platform (HCA DCP).
+
+## Definitions
+
+__Community pipeline:__ A pipeline that is not officially used as a standard in the HCA project but runs in the DCP and is operated by the Data Processing Service to enable community members to work with a diversity of submitted data until they have a standardized pipeline. These pipelines are not as vetted as standardized pipelines.  
+__HCA Analysis Working Group (AWG):__ A group of domain specialists that advise on and monitor scientific choices made in the Human Cell Atlas Project.  
+__HCA DCP Governance Group:__ A working group of the Human Cell Atlas project that advises the DCP, assuring alignment of DCP development efforts with the scientific project it supports.  
+__On-boarding:__ The process of taking an existing pipeline and engineering it to work in the HCA DCP.  
```

Comment:
nit: [Onboarding](https://en.wikipedia.org/wiki/Onboarding) is not hyphenated.

Reply:
thanks!